### PR TITLE
Listing created scopes sorted by name

### PR DESCRIPTION
**Pull request by AmaliMatharaarachchi**
Fix https://github.com/wso2/product-apim/issues/6216

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS2Parser.java [hunk 1]
```diff
@@ -215,6 +216,12 @@
         return scopeList;
     }
 
+    public Set<Scope> sortScopes(Set<Scope> scopeSet){
```
Comment (ruks) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS2Parser.java]: do we need to make this method public?

Comment (ruks) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS2Parser.java]: better to add method level comments.

Comment (AmaliMatharaarachchi) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS2Parser.java]: Fixed. Thanks.

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java [hunk 2]
```diff
@@ -181,12 +182,25 @@
...
     }
 
+    /**
+     * Sort scopes by name.
+     * This method was added to display scopes in publisher in a sorted manner.
+     *
+     * @param scopeSet
+     * @return Scope set
+     */
+    private Set<Scope> sortScopes(Set<Scope> scopeSet) {
```
Comment (isharac) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: better move to a utility class as same method is duplicated.

Comment (AmaliMatharaarachchi) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/definitions/OAS3Parser.java]: Fixed in https://github.com/wso2/carbon-apimgt/pull/8244

**Pull request merged**